CHANGELOG.md

```diff
@@ -46,6 +46,7 @@ For more information on this release, see:
 - Changed two scripts of the main Handsontable workspace (`./`), to speed up the build process: now, the `npm run build` and `npm run test` scripts don't build or test the Handsontable examples (`./examples`). [#9412](https://github.com/handsontable/handsontable/issues/9412)
 - Changed the version of the Moment.js dependency from 2.24.0 to 2.29.3, in reaction to a recently-found Moment.js security vulnerability that did not directly affect Handsontable. [#9381](https://github.com/handsontable/handsontable/issues/9381)
 - Changed the version of the HyperFormula optional dependency from ^1.2.0 to ^2.0.0. [#9411](https://github.com/handsontable/handsontable/issues/9411)
+- Changed an internal property name, from `hot.view.wt` to `hot.view._wt`, to make it clear that Walkontable (Handsontable's rendering engine) is not a part of Handsontable's public API. [#8760](https://github.com/handsontable/handsontable/issues/8760)
```

Comment:
It is not clear what `hot` is (it is an instance of the Handsontable class).

There really should be a docs page about it. Actually I am talking about the same thing in the comment for cell D38 in https://docs.google.com/spreadsheets/d/1FeUc6uJqnVqAoDF3fSRDTEWM-UQqje6_ZEyIGTFEG_U/edit#gid=832314621. We must create a new page in the docs called "Working with Handsontable APIs" which explains that all public APIs are exposed as methods and properties of an instance created by the Handsontable class constructor. Maybe you can create an issue for it and add it to the #9303 epic?

As a short term solution, I recommend to change the above text to:

Changed an internal property name, from `hot.view.wt` to `hot.view._wt`, where `hot` is the name of your Handsontable instance. The intention of this change is to make it clear that Walkontable (Handsontable's rendering engine) is not a part of Handsontable's public API. [#8760](https://github.com/handsontable/handsontable/issues/8760)

-----

Alternatively, make it make it self-descriptive by changing all references to `hot` in this PR to `handsontableInstance`.

Reply:
> Alternatively, make it make it self-descriptive by changing all references to hot in this PR to handsontableInstance.

@warpech I think that's the best solution, which should be applied throughout the docs.

Done for this PR: [64d92a4](https://github.com/handsontable/handsontable/pull/9550/commits/64d92a468025af7ec35fa0f6f08bba26dd31972e)

Issue created: #9584